baselines/asset/protos/google/cloud/asset/v1/asset_service.proto.baseline

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 Google LLC
```

Comment:
This is a little strange that it changes the date?

Reply:
The original protos are (c) 2020 and apparently someone changed _everything_ to 2021 even though this particular change was not needed.